File: BackEnd/Data/Entities/Game.cs
namespace BackEnd.Data.Entities
{
    public class Game
    {
        public int Id { get; set; }
        public string GameName { get; set; }
        public string CompanyName { get; set; }
        public double Price { get; set; }
        public string Summary { get; set; }
        public DateTime ReleaseDate { get; set; }
    }
}

File: front-end2/src/types/game.ts
export interface BasicGame{
    id:number,
    gameName:string,
    companyName:string,
    price:number,
    releaseDate:  Date
    
}
export interface Game extends BasicGame{
    summary:string
}

export interface CreateGame{
    gameName:string | undefined,
    companyName:string | undefined,
    price:number | undefined,
    summary:string | undefined,
    releaseDate:  Date | undefined,
}